<?xml version="1.0" ?>
<launch xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="https://gist.githubusercontent.com/nalt/dfa2abc9d2e3ae4feb82ca5608090387/raw/roslaunch.xsd">
    <arg name="serial_path" default="~/.config/niryo/robot_serial_do_not_touch"/>

    <node name="niryo_robot_serial_number" pkg="niryo_robot_iot" type="serial_number_node.py" output="screen">
        <param name="serial_path" value="$(arg serial_path)"/>
    </node>
</launch>
